<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RentalDocumentWidget</class>
 <widget class="QWidget" name="RentalDocumentWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>891</width>
    <height>737</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="mainVerticalLayout" stretch="0,0,1">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="toolbarWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>55</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>55</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="toolBarHorizonLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="statusFrame">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="statusHorizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget" native="true"/>
      </item>
      <item>
       <widget class="QWidget" name="radioWidget" native="true">
        <layout class="QHBoxLayout" name="radioHorizonLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QRadioButton" name="reservationRadio">
           <property name="text">
            <string>0.预约</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="unconfirmedRadio">
           <property name="text">
            <string>1.已确认</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="confirmedRadio">
           <property name="text">
            <string>2.未确认</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="totalRadio">
           <property name="text">
            <string>3.全部</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_3" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="documentWidget" native="true">
     <layout class="QHBoxLayout" name="documentHorizonLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QTreeWidget" name="clientTreeWidget">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <column>
         <property name="text">
          <string>1</string>
         </property>
        </column>
        <item>
         <property name="text">
          <string>新建项目</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>新建项目</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>新建项目</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="docTableWidget">
        <row>
         <property name="text">
          <string>新建行</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>新建行</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>新建行</string>
         </property>
        </row>
        <column>
         <property name="text">
          <string>承租单位</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>工程名称</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>施工部位</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>泵式</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>混凝土标号</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>车牌型号</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>施工日期</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>进场时间</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>离场时间</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>工作时长</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>泵送方量(m3)</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>新建列</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>开工燃油</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>工程总价</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>现场负责人</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>负责人联系电话</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>当班司机1</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>当班司机2</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>当班司机3</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>备注</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
